frontend/src/styles/material.sass

```diff
@@ -42,3 +42,22 @@ snack-bar-container
         text-align: center
     & span
         width: 100%
+
+.mat-form-field-underline
+    display: none
+
+.mat-form-field-appearance-fill .mat-form-field-infix
+    padding: 0
+
+.mat-form-field-appearance-fill .mat-form-field-flex
+    background-color: $white
+    padding: 0
+
+.mat-form-field-wrapper
+    padding-bottom: 5px
+
+.mat-select-value
+    margin-bottom: 5px
+
+th
+    height: 20px
```

Comment:
Probably you move this accidentally

Reply:
yes, I realized that I don't need it at all